UNITTESTS/target_h/platform/mbed_retarget.h

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
That doesn't look right. The header guard is missing and many declarations are missing.

Reply:
Added include guards. Some platform code was stubbed because there were few definition conflicts with GNU C++ standard library.